test/Application.UTest/Common/Services/CompetitiveRatingModelTest.cs

```diff
@@ -7,12 +7,11 @@ namespace Crpg.Application.UTest.Common.Services;
 public class CompetitiveRatingModelTest
 {
     [Test]
-    [Ignore("Broken for now")]
     public void HigherRatingGivesHigherCompetitiveRating()
     {
         CompetitiveRatingModel competitiveRatingModel = new();
         float cr1 = competitiveRatingModel.ComputeCompetitiveRating(
-            new CharacterRating { Value = 100, Deviation = 10 });
+            new CharacterRating { Value = 100, Deviation = 500 });

```

Comment:
The test becomes confusing to me, if they don't have the same deviation I have no idea which one should have the highest competitive rating.